index.js

```diff
@@ -123,24 +141,43 @@ datasetsToTest.forEach(async dataset => {
         const previewUrl = resource.path
         const resPreview = await fetch(previewUrl)
         statuses.csv_preview_links = resPreview.status + ': ' + resPreview.statusText
-      } 
+      } else if (!statuses.csv_preview_links){
+        statuses.csv_preview_links ='NOT GENERATED'
+      }
     }
     
     // datapackage_json
     const datapackageLink = $('.container').find('.btn-default')
     if ( datapackageLink.text() === 'Datapackage.json') {
       let datapackageUrl = baseUrl + datapackageLink[0].attribs.href
-      const dpUrl = await resourceLinks(datapackageUrl)
+      const dpUrl = await checkPage(datapackageUrl)
       statuses.datapackage_json = dpUrl.status + ':' + dpUrl.statusText
     }
-  }
+    // page loading time 
+    const loadTime = await pageLoadTime(showcaseUrl)
+    statuses.total_load_time = loadTime.TaskDuration
+    statuses.script_laod_time = loadTime.ScriptDuration
+    statuses.layout_load_time = loadTime.LayoutDuration
+    
+    // graphs
+    const graphC = $('.svg-container').css()
+    if (graphC.position === 'relative' && graphC.width === '670px' && graphC.height === '450px') {
```

Comment:
are you sure this check is enough? is this always the case, Eg what if we change dataset?

Reply:
@zelima @anuveyatsu suggested to test this part, it is fixed